index.html

```diff
@@ -0,0 +1,65 @@
+<!DOCTYPE html>
+	<html>
+	<head>
+		<link rel="stylesheet" type="text/css" href="css/styles.css">
+		<title>KANYE RESUME</title>
+	</head>
+
+	<body>
+		<nav></nav>
```

Comment:
:hammer: As I explained in the comments for your css, best practice would say not to only use semantic tags, such as `<nav>` for what they were made for. It would be preferred to use a `<div>` here instead and style it how you'd like.